docs/usage.md

```diff
@@ -77,55 +70,39 @@ Before performing the steps outlined in this section, we recommend setting the g
+    1. **Outbreak Alerts**: The outbreak alert contain the following
 
         - Outbreak Alert details
 
         - Mitigation Details
 
-    2. Threat Hunt Rules: The following rules are created with the solution pack:
+    2. **Threat Hunt Rules**: The following rules are imported with outbreak-specific response solution pack:
 
-        - Yara Rules: YARA Rules help malware researchers identify and classify malware samples that focus on scanning and identifying malicious files.
+        - **Yara Rules**: YARA Rules help malware researchers identify and classify malware samples that focus on scanning and identifying malicious files.
 
         ![Yara Rules](./res/yara_rule.png)
 
-        - Sigma Rules: Sigma Rules provide a standard format for log events. They are helpful in searching or pattern matching through log data.
+        - **Sigma Rules**: Sigma Rules provide a standard format for log events. They are helpful in searching or pattern matching through log data.
         
-            Using Sigma Rules we can create signature-based Threat Hunt Rules for various Threat Detection Integrations like SIEM, Analyzer, or EDR.
+        Using Sigma Rules we can create signature-based Threat Hunt Rules for various Threat Detection Integrations like SIEM, Analyzer, or EDR.
         
-        - Fortinet Fabric Rules: With every Outbreak Alert FortiGuard provides FortiAnalyzer and FortiSIEM as part of Fortinet fabric solution.
+        - **Fortinet Fabric Rules**: With every Outbreak Alert FortiGuard provides FortiAnalyzer and FortiSIEM as part of Fortinet fabric solution.
 
-9. Head over to the **Dashboard** and select **Outbreak Response Overview** to view following information.
+9. Select the **Dashboard** tab, under *Outbreak Management*, to view following information:
 
     ![Outbreak Dashboard](./res/dashboard-outbreak-response-overview.png)
 
-## Workaround - Uniqueness Constraint Violation
-
-Under demo mode when *Threat Hunting - FortiAnalyzer - Get Related Assets* playbook creates an asset, a uniqueness constraint violation occurs. This section offers a workaround.
-
-1. Navigate to **Automation** > **Playbooks**.
-
-2. Click to open the **10 - SP - Outbreak Response Framework** playbook collection.
-
-3. Click to open the **Threat Hunting - FortiAnalyzer - Get Related Assets** playbook.
-
-4. Double-click to open **Create Asset** step.
-
-5. Click the Correlations tab under **Fields**.
-
-6. Select **Overwrite** and then **Append**.
-
-7. Click **Save** and **Save Playbook** to save the changes to the playbook.
-
 # Next Steps
 
 | [Installation](./setup.md#installation) | [Configuration](./setup.md#configuration) | [Contents](./contents.md) |
-|-----------------------------------------|-------------------------------------------|---------------------------|
+|-----------------------------------------|-------------------------------------------|---------------------------|
```

Comment:
Need to update known issues after upgrade:
1. Need to add "Deactivated" status to outbreak alert status.
2. Edit the step "Update Last Pull Time" field " status to "tracking" in "> Investigate Outbreak Alert" PB.
3. In Outbreak Alert Time Frame Analysis last step "Update Outbreak Alert Status" user needs to select status as "deactivated"
4.